tools/sdk-generation-pipeline/documents/docker/vscode-connect-docker-container.md

```diff
@@ -20,8 +21,4 @@ Then you can write your codes in vscode.
 
     Answer: Vscode C# extension is based on OmniSharp, which sometimes make us confused. To resolve it:
     1. Run `dotnet build` to rebuild the project
-    2. In vscode, press `ctrl + shift + p` and then type `Restart Omnisharp`.
-
-2. Vscode cannot load java project correctly.
-
-    Answer: Java repository is very large and vscode extension for Java needs to spend much time to load the project. You can check the progress in the terminal of `Java Build Status`.
\ No newline at end of file
+    2. In vscode, press `ctrl + shift + p` and then type `Restart Omnisharp`.
```

Comment:
I think this command is for windows only,

Reply:
I change it to `F1` and believe it applys to all os.